src/or/include.am

```diff
@@ -254,11 +263,22 @@ ORHEADERS = \
 	src/or/routerset.h				\
 	src/or/routerparse.h				\
 	src/or/scheduler.h				\
+	src/or/shared_random_common.h			\
 	src/or/statefile.h				\
 	src/or/status.h					\
 	src/or/torcert.h				\
 	src/or/tor_api_internal.h
 
+# We add the headers of the modules even though they are disabled so we can
+# properly compiled the entry points replacedment.
```

Comment:
I think `replacedment` is a typo here. Maybe 'stub functions' are better?

Reply:
Yes!